Restructured and updated docs (#36)
IDinsight/aaq-core: docs/index.md

<pre align="center" style="text-align:center; font-size: 1vw; background:none;">
    __            __        ______
    /""\          /""\      /    " \
    /    \        /    \    // ____  \
    /' /\  \      /' /\  \  /  /    )  )
  //  __'  \    //  __'  \(: (____/ //
 /   /  \\  \  /   /  \\  \\         \
 (___/    \___)(___/    \___)\"____/\__\
ASK-A-QUESTION

</pre>

**[Ask-a-question](https://idinsight.github.io/aaq-core/) is an open-source product designed to support NGOs,

> developing country governments, and social sector orgs to use Large Language Models to answer questions from

Reviewer: ```suggestion
governments of developing countries, and social sector orgs to use Large Language Models to answer questions from
```

citizens.**

---

## :woman_cartwheeling: Features

**:question: LLM-powered search** : Match your questions to content in the database using embeddings from LLMs.

**:robot: LLM responses** : Craft a custom reponse to the question using LLM chat and the content in your database

> **:speech_balloon: Deploy on whatsapp** : Easily deploy using WhatsApp Business API

Reviewer: ```suggestion
**:speech_balloon: Deploy on WhatsApp** : Easily deploy using WhatsApp Business API
```

**:books: Manage content** : Use the Admin App to add, edit, and delete content in the database

---

## :construction: Upcoming

> **Step 3:** Run docker-compose

Reviewer: ```suggestion

```

> **Step 3:** Run docker-compose

Reviewer: ```suggestion
```

Reviewer: Left over from previous version

> **:earth_africa: Support for low resourced language**: Ask questions in local languages. Languages currently on the roadmap
> 
> - [ ] Xhosa
> - [ ] Zulu
> - [ ] Hindi
> - [ ] Igbo
> 
> **:speech_balloon: conversation capability**: Refine or clarify your question through conversation
> 
> **:video_camera: multimedia content**: Respond with not just text but images and videos as well.
> 
> **:rotating_light: urgency detection**: Identify and handle urgent messages.
> 
> **:woman_technologist: engineering dashboard**: Monitor uptime, response rates, throughput HTTP reponse codes and more
> 
> **:woman_office_worker: content manager dashboard**: See which content is the most sought after, the kinds of questions that receive poor feedback, identify missing content, and more

Reviewer: ```suggestion
**:earth_africa: Support for low resourced language**: Ask questions in local languages. Languages currently on the roadmap:

- [ ] Xhosa
- [ ] Zulu
- [ ] Hindi
- [ ] Igbo

**:speech_balloon: Conversation capability**: Refine or clarify your question through conversation

**:video_camera: Multimedia content**: Respond with not just text but images and videos as well.

**:rotating_light: Urgency detection**: Identify and handle urgent messages.

**:woman_technologist: Engineering dashboard**: Monitor uptime, response rates, throughput HTTP reponse codes and more

**:woman_office_worker: content manager dashboard**: See which content is the most sought after, the kinds of questions that receive poor feedback, identify missing content, and more.
```

Reviewer: Standardised capitalisation and added some punctuation

!!! question "Looking for other features?"

>     If you are an NGO or a social sector organisation, we'd love to hear what features you'd

Reviewer: /nit seems odd to leave out government in this given that we said it was for ngos, social orgs, and governments in the intro haha

Author: Good point. Added!

    like to see. Raise an issue with `[FEATURE REQUEST]` before the title to start the conversation.

---

## Acknowledgements

The project is funded by Google.org through the

> [AI for Global Goals](https://globalgoals.withgoogle.com/globalgoals/) grant.

Reviewer: I quite liked their logo being here - did they object?

Author: Good idea. Added!
